### PR TITLE
modified + added docstrings

### PR DESCRIPTION
**Pull request by sandip-baidya**
1. Modified existing docstrings
2. Added new docstrings along with code snippets
3. Code snippets are working correctly

### CODE REVIEW

File: docs/codes-rst/natural_photo_se_text.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+NaturalPhotoShabd
+#########################
+
+.. module:: freshlybuiltimagebol.natural_photo_se_text
+
+.. autoclass:: NaturalPhotoShabd
+
```
Comment (devkapilbansal) [inline docs/codes-rst/natural_photo_se_text.rst:7]: Can you please add data in it?

File: freshlybuiltimagebol/text_bol_uthega.py [hunk 2]
```diff
@@ -5,35 +5,95 @@
...
+        :type bhasha: string
+        :returns: translated text
+        :rtype: string
+        
+        >>> from freshlybuiltimage.text_bol_uthega import ShabdDhwani
+        >>> print(ShabdDhwani.shabd_ki_bhasha_badlo("I am so happy!", 'french'))
+        Je suis très heureux.
+
+        .
+
```
Comment (devkapilbansal) [inline freshlybuiltimagebol/text_bol_uthega.py:61]: Why so many blank lines here?

File: freshlybuiltimagebol/text_bol_uthega.py [hunk 3]
```diff
@@ -5,35 +5,95 @@
...
+        :param bhasha: Language that you want the audio to be in.
+        :type bhasha: string
+        :param filename: path/to/filename.mp3 file name must be fed as a string. For e.g. ``shabd_se_dhwani("I am happy", "english", "happy.mp3")`` will save an audio file called ``happy.mp3`` in the same folder as your python script.
+        :type filename: string
+
+        >>> from freshlybuiltimage.text_bol_uthega import ShabdDhwani
+        >>> import os
+        >>> ShabdDhwani.shabd_se_dhwani("This is a wonderful world.", "english", "trans.mp3")
+        >>> os.startfile("trans.mp3")
+
```
Comment (devkapilbansal) [inline freshlybuiltimagebol/text_bol_uthega.py:35]: Here too, we shouldn't leave extra blank lines

File: freshlybuiltimagebol/text_bol_uthega.py [hunk 4]
```diff
@@ -5,35 +5,95 @@
...
         return bhasha
 
     def shabd_se_dhwani(shabd, bhasha, filename):
-        """ shabd = text,
-            bhasaa = language(mainly allowed in code)
-                = file
-            filename=app//audio.mp3
-            anuvadak = Translator
-            anuvadit = Translated
+        """Listen to text in audio, in a specific language.
```
Comment (devkapilbansal) [inline freshlybuiltimagebol/text_bol_uthega.py:20]:
```suggestion
        """
        Listen to text in audio, in a specific language.
```

File: freshlybuiltimagebol/text_bol_uthega.py [hunk 5]
```diff
@@ -5,35 +5,95 @@
 
 
 class ShabdDhwani:
+    """A class used for translation and audio generation of texts.
```
Comment (devkapilbansal) [inline freshlybuiltimagebol/text_bol_uthega.py:8]: Break it like this
```suggestion
    """
    A class used for translation and audio generation of texts.
```

File: docs/index.rst [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+   references
+
+
+
+Indices and tables
+==================
+
+* :ref:`genindex`
+* :ref:`modindex`
+* :ref:`search`
```
Comment (devkapilbansal) [inline docs/index.rst:28]: These links are going nowhere. Is it intential?

File: docs/codes-rst/text_bol_uthega.rst [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+shabd_ki_bhasha_badlo
+**********************
+
+.. automethod:: ShabdDhwani.shabd_ki_bhasha_badlo
+
+
+
+
+
```
Comment (devkapilbansal) [inline docs/codes-rst/text_bol_uthega.rst:31]: Remove these blank lines too

**Review by devkapilbansal — CHANGES_REQUESTED** [review]
@sandip-baidya can you please look into these

File: freshlybuiltimagebol/text_bol_uthega.py (hunk 5 above)
Comment (sandip-baidya) [reply freshlybuiltimagebol/text_bol_uthega.py:8]: doesn't really make any difference, but I'll do that if it increases readability.

**Review by devkapilbansal — APPROVED** [review]
LGTM :+1: 

Although there are some corrections that can be done in another pull request

**Pull request merged**